microsoft/react-native-test-app
# fix(Windows): Support autolinking

Resolves #219

#### Test Plan

1. Bump `react-native` and `react-native-windows` to ^0.63
2. Add a 0.63 compatible package with Windows support, e.g. `react-native-camera`
3. Run `yarn`
4. Run `yarn install-windows-test-app`
5. Open `windows/Example.sln`
6. Observe that `AutolinkedNativeModules.g.cpp` contains your package
7. Build and run

```diff
diff --git a/example/package.json b/example/package.json
index d27ceb8..800f7fb 100644
--- a/example/package.json
+++ b/example/package.json
@@ -21,10 +21,10 @@
   "devDependencies": {
     "@babel/core": "^7.0.0",
     "mkdirp": "^0.5.1",
-    "react": "16.11.0",
-    "react-native": "0.62.2",
-    "react-native-macos": "0.62.10",
+    "react": "16.13.1",
+    "react-native": "0.63.2",
+    "react-native-camera": "^3.39.1",
     "react-native-test-app": "../",
-    "react-native-windows": "0.62.9"
+    "react-native-windows": "0.63.2"
   }
 }
```
